src/project-2.js

```diff
@@ -44,62 +81,102 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+
+  if (num === 0 || num === 1) {
+    return false;
+  }
+
+  for (let i = num - 1; i > 1; i--) {
+    if (num % i === 0) {
+      return false;
+    }
+  }
+
+  return true;
 };
 
 const returnFirst = (arr) => {
   // return the first item from the array
+  return arr.shift();
 };
 
 const returnLast = (arr) => {
   // return the last item of the array
+  return arr.pop();
 };
 
 const getArrayLength = (arr) => {
   // return the length of the array
+  return arr.length;
 };
 
 const incrementByOne = (arr) => {
   // arr is an array of integers
   // increase each integer by one
   // return the array
+  const incrementedArray = arr.map(element => element += 1);
+
+  return incrementedArray;
 };
 
 const addItemToArray = (arr, item) => {
   // add the item to the end of the array
   // return the array
+  arr.push(item);
+
+  return arr;
 };
 
 const addItemToFront = (arr, item) => {
   // add the item to the front of the array
   // return the array
   // hint: use the array method .unshift
+  arr.unshift(item);
+
+  return arr;
 };
 
 const wordsToSentence = (words) => {
   // words is an array of strings
   // return a string that is all of the words concatenated together
   // spaces need to be between each word
   // example: ['Hello', 'world!'] -> 'Hello world!'
+
+  return words.join(' ');
 };
 
 const contains = (arr, item) => {
   // check to see if item is inside of arr
   // return true if it is, otherwise return false
+  const elementPresent = arr.indexOf(item);
```

Comment:
Opportunity for refactor - One-liner

```javascript
const contains = (arr, item) => arr.indexOf(item) >= 0;
```

Reply:
Used `includes` instead:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

```
const contains = (arr, item) => arr.includes(item);
```

Reply:
`includes` is fine for the exercise, but we will still be using `indexOf` for a long time in production (as it is not supported in our favorite browser as of yet).